gen/templates/resource.go

```diff
@@ -665,6 +816,230 @@ func (r *{{camelCase .Name}}Resource) ImportState(ctx context.Context, req resou
 
 	helpers.SetFlagImporting(ctx, true, resp.Private, &resp.Diagnostics)
 }
+{{- end}}
+
+{{- if .IsBulk}}
+func (r *{{camelCase .Name}}Resource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	// Import looks for string in the following format: Domain,[ObjectName1,ObjectName2,...]
+	// Domain part is optional
+	// ObjectName1,ObjectName2,... is coma-separated list of object names
+	var config {{camelCase .Name}}
+
+	// Compile pattern for import command parsing
+	var inputPattern = regexp.MustCompile(`^(?P<domain>[^\s,]*),*\[(?P<names>.*?),*\]`)
+
+	// Parse parameter
+	match := inputPattern.FindStringSubmatch(req.ID)
+
+	// Check if regex matched
+	if match == nil {
+		resp.Diagnostics.AddError("Import error", "Failed to parse import parameters")
```

Comment:
Can we maybe also include a hint on how it should look like? Something like `<domain_name>,[<object1_name>,<object2_name>,...]`.

Reply:
Added clarification to the error message.